src/providers/historical-telemetry-provider.js

```diff
@@ -27,61 +27,55 @@ import {
 } from '../utils.js';
 
 export default class YamcsHistoricalTelemetryProvider {
-    constructor(url, instance) {
+    constructor(url, instance, openmct) {
```

Comment:
just being too picky, prefer, `openmct` as first param. 😅

Reply:
take a look at my commit message you picky person... 💯